src/components/StickyBanner.jsx

```diff
@@ -0,0 +1,75 @@
+import styled from '@emotion/styled';
+
+const StickyBannerContainer = styled.div`
+  height: 96px;
+  width: 768px;
+  position: absolute;
+  bottom: 0;
+  right: 25%;
+  left: 50%;
+  margin-left: -384px;
```

Comment:
Based on Figma, the sticky banner should have a white-background wrapper that spans the entire width of the application, and its inner content should be constrained to the width of `theme.layout.maxWidth` and centered.

Can you please update the style in the Desktop and Tablet views such that it matches what is illustrated in the Figma design? 

Key points to implement:
1. Full-width white background
2. Centered content with max-width of `theme.layout.maxWidth`
3. Ensure the banner remains sticky at the bottom of the viewport

See the screenshot below for a visual comparison between the current implementation and the Figma design.

**Local Environment**
![Screenshot 2024-07-25 at 2 27 52 PM](https://github.com/user-attachments/assets/3f3ec448-629a-4be3-99ac-26b142ccd0fb)

**Figma Design**
![Screenshot 2024-07-25 at 2 24 42 PM](https://github.com/user-attachments/assets/87a94dc2-66ca-4fe1-b979-5a50a6d898be)